scss/layout/_grid.scss

```diff
@@ -68,6 +69,16 @@
   }
 }
 
+.slab--primary .grid {
+  a {
```

Comment:
I think we should consider putting either these link styles in their own class or making a class for these slabs on the home page. Elsewhere, on a dark background, we use white links. This would make it so that any time a link is in a `.grid` container within a `.slab` the links would be a color. 

@jenniferthibault do you have thoughts on what the rules ought to be for when a link is white vs a contrasting color?